components/eam/src/physics/rrtmgp/radiation.F90

```diff
@@ -617,6 +624,28 @@ subroutine radiation_init(state,pbuf)
       call add_hist_coord('swband', nswbands, 'Shortwave wavenumber', 'cm-1', sw_band_midpoints)
       call add_hist_coord('lwband', nlwbands, 'Longwave wavenumber', 'cm-1', lw_band_midpoints)
 
+
+      ! Add outfields to be used for RRTMGP stand-alone testing.
```

Comment:
General comment, probably no need to change this since it appears to be pretty ubiquitous in E3SM, but these do not appear to be used in this routine, only later when/if these are actually written out. I kind of prefer to keep this query closer to its use, it makes for more readable code, unless this saves computational cost by only initializing once.